examples/oidc_cli/src/main.rs

```diff
@@ -280,9 +281,33 @@ impl OidcCli {
         let StoredSession { client_session, user_session, client_credentials } =
             serde_json::from_str(&serialized_session)?;
 
+        println!("homeserver = {}", client_session.homeserver);
```

Comment:
Since this is an example, it would be nice to document why we need to use auto-discovery even when restoring the session